gammapy/data/pointing.py

```diff
@@ -272,22 +273,22 @@ def from_fits_header(cls, header):
 
     def to_fits_header(self, format="gadf", version="0.3", time_ref=None):
         """
-        Convert this FixedPointingInfo object into a fits header for the given format
+        Convert this FixedPointingInfo object into a fits header for the given format.
 
         Parameters
         ----------
-        format : str
-            Format, currently only "gadf" is supported
-        version : str
+        format : str, optional
+            Format, currently only "gadf" is supported. Default is "gadf".
+        version : str, optional
             Version of the ``format``, this function currently supports
-            gadf versions 0.2 and 0.3.
-        time_ref : astropy.time.Time or None
-            Reference time for storing the time related information in fits format
+            gadf versions 0.2 and 0.3. Default is "0.3".
```

Comment:
```suggestion
            gadf versions 0.2 and 0.3. Default is 0.3.
```
In my opinion it is better to use "" only for strings.

Reply:
It is actually a string. 
`def to_fits_header(self, format="gadf", version="0.3", time_ref=None):`